packages/versioning/src/lib.ts

```diff
@@ -63,6 +63,13 @@ const libDef = {
         versionedDependencyRemovedBefore: paramMessage`'${"sourceName"}' is referencing type '${"targetName"}' removed in version '${"targetAddedOn"}' but version used is ${"dependencyVersion"}.`,
       },
     },
+    "incompatible-versioned-namespace-use-dependency": {
+      severity: "error",
+      messages: {
+        default:
+          "The useDependency decorator can only be used on a Namespace if the namespace is unversioned.",
```

Comment:
Maybe worth hinting at the fix here, to apply it to the version enum instead, right?

Reply:
Updated the message.